app/models/services/pubmed/resource.rb

```diff
@@ -33,40 +22,24 @@ def map_attributes mapper, data
 
     def mapper
       {
-        title:              lambda {|data| data.xpath('//ArticleTitle').text },
-        publication:        lambda {|data| data.xpath('//Journal/Title').text },
-        doi:                lambda {|data| data.xpath('//ELocationID').text },
-        pubmed_id:          lambda {|data| data.xpath('//PMID').text },
-        abstract:           lambda {|data| data.xpath('//AbstractText').map(&:text).join("\n\n") },
-        published_at:       lambda {|data| Date.parse data.xpath('//PubDate').map(&:text).join(' ') },
-        authors_attributes: lambda {|data| authors = data.xpath('//AuthorList/Author')
+        title:              lambda {|data| data.css('ArticleTitle').text },
+        publication:        lambda {|data| data.css('Journal Title').text },
+        doi:                lambda {|data| data.css('ArticleId[IdType=doi]').text },
+        pubmed_id:          lambda {|data| data.css('PMID').text },
+        abstract:           lambda {|data| data.css('AbstractText').map(&:text).join("\n\n") },
+        published_at:       lambda {|data| Date.parse data.css('PubDate').map(&:text).join(' ') },
```

Comment:
I also think it might be able to optimize the xpath query a bit, because I don't know how to use xpath. Apparently using `//SoAndSo` is inefficient because it searches the whole document for the node. If I really wanted it to be fast, I would use an absolute path like I had before - `//ArticleSet/Article/etc...` - but it's just so ugly to look at...